internal/cli/atlas/clusters/update.go

```diff
@@ -85,6 +85,13 @@ func (opts *UpdateOpts) cluster() (*atlasv2.AdvancedClusterDescription, error) {
 		if opts.name == "" {
 			opts.name = cluster.GetName()
 		}
+
+		removeReadOnlyAttributes(cluster)
```

Comment:
do you still need the call in line 101?

Reply:
We have this piece of code that prevents us from running the function for filename. Actually, Let me remove it 

```golang
	if opts.filename == "" {
		opts.patchOpts(cluster)
	}
```

Reply:
updated it